config/docker/thinking_sphinx.yml

```diff
@@ -5,20 +5,12 @@ common: &sphinx
   html_remove_elements: "style, script"
   html_index_attrs: "img=alt,title; a=title"
   min_infix_len: 3
-  enable_star: 1
```

Comment:
what is this?

Reply:
This is an option removed in sphinx `2.2.X`